.copr/Makefile

```diff
@@ -0,0 +1,50 @@
+# Copr does not have Git available. Install it before anything else.
+INSTALLER := $(shell [ -f /usr/bin/dnf ] && echo dnf || echo yum)
+INSTALL_GIT := $(shell [ -f /usr/bin/git ] || $(INSTALLER) -y install git)
+
+NAME = product-listings-manager
+VERSION := $(shell git describe --tags --abbrev=0 --match 'v*' | sed 's/^v//')
+COMMIT := $(shell git rev-parse HEAD)
+SHORTCOMMIT := $(shell echo $(COMMIT) | cut -c1-7)
+RELEASE := $(shell git describe --tags --match 'v*' \
+             | sed 's/^v//' \
```

Comment:
No need for this line.

Reply:
I'm not sure I understand. What change are you suggesting for this part?

Reply:
Dropping the `v` from version string (`sed 's/^v//'`) is not really needed since this will get only the release part.

Reply:
Doh, thanks! I see now. I've dropped this line and re-pushed.